### ISSUE TITLE
Pressing "new track" link on homepage can result in NoMethodError

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/v3-beta#364]
![image](https://user-images.githubusercontent.com/1964376/131659043-d5803da8-6202-4fae-9f6b-ad4695f1fa30.png)

![image](https://user-images.githubusercontent.com/1964376/131659103-c03e95d5-2b4d-4108-9e64-38eef5257779.png)

Maybe it's because I've joined but not finished Hello World!?

![image](https://user-images.githubusercontent.com/1964376/131659813-b1ba0bea-d9a4-4d62-9552-618447de7a9b.png)

**Comment by iHiD** [comment]
I think it's because it's a WIP exercise.

**Comment by iHiD** [comment]
@ErikSchierboom I think this got done?

**Comment by ErikSchierboom** [comment]
Yes. The proper fix is still a draft PR: https://github.com/exercism/website/pull/1659 I haven't gotten around finishing that.
